lib/timed_controller.dart

```diff
@@ -96,7 +101,7 @@ class TimedController extends WidgetsBindingObserver {
   fetchData() async {
     try {
       //Fetch the ticks
-      int tick = await _apiService.getCurrentTick();
+      int tick = (await _archiveApi.getCurrentTick()).tick;
```

Comment:
tick-data should be used
latestTick in the archiver returns the latest tick processed by the archiver, not the current tick in the network (best case scenario might be a difference of one tick less in the archiver)